constrain/library/fan_static_pressure_reset_control.py

```diff
@@ -1,67 +1,31 @@
 from constrain.checklib import RuleCheckBase
```

Comment:
If you could, we probably also want the docstrings like other verifications.

Reply:
Addressed.